MoneySplitter.Win10/ViewModels/FriendsViewModel.cs

```diff
@@ -28,25 +28,25 @@ public FriendsViewModel(IFriendsManager friendsManager)
             _friendsManager = friendsManager;
         }
 
-        protected override void OnActivate()
-        {
-            base.OnActivate();
-            Friends = new ObservableCollection<UserModel>(_friendsManager.FriendsOfCurentUser);
-        }
-
-        public async Task RemoveFriendAsync(int idFriend)
+        public async Task RemoveFriendAsync(int friendId)
         {
-            var friend = Friends.Where(x => x.Id == idFriend).First();
+            var friend = Friends.Where(x => x.Id == friendId).First();
             Friends.Remove(friend);
 
-            var isSuccessResponce = await _friendsManager.RemoveFriendAsync(idFriend);
+            var isSuccessResponce = await _friendsManager.RemoveFriendAsync(friendId);
 
             if (isSuccessResponce)
             {
-                await _friendsManager.LoadCurrentFriendsUserAsync();
+                await _friendsManager.LoadUserFriendsAsync();
             }
         }
 
+        protected override void OnActivate()
+        {
+            base.OnActivate();
+            Friends = new ObservableCollection<UserModel>(_friendsManager.UserFriends);
```

Comment:
I recommend to use async method GetUserFriendsAsync because in your case UserFriends can be null e.g. they not loaded at this moment